pages/in/widget/iscn/index.vue

```diff
@@ -279,6 +280,7 @@ export default {
       opener: opener && opener !== '0',
       state,
       url,
+      blocking,
```

Comment:
should check with same logic as `opener`

Reply:
ok! added on next commit